### ISSUE TITLE
🐛 BUG: Next pages of bookmarks load inconsistently at higher resolutions

### ISSUE THREAD

**Issue by rodrigojmr** [dailydotdev/daily#243]
### What went wrong? 🤔

Because I use a *1440p screen, starting at around 2000px width, bookmarks sometimes don't load the next page. Because of this I was afraid I had lost all my bookmarks, but they showed on my laptop. At 2560px, it almost always does not load.

[Video](https://streamable.com/c7f1da):

### Expected Behavior

For bookmarks to load consistently.

### Actual Behavior

The wider the resolution gets, the less likely the next pages of bookmarks are to be loaded.

### Steps to Reproduce Issue

Expand window to 2560px width.

### Solution

Shrink window back to 1080px width.

#### Operating System
Windows 10 Version Build 19042

#### Version of Daily
Daily: 2.29.0

#### Browser
Chrome Version 87.0.4280.141

**Comment by idoshamun** [comment]
Hey @rodrigojmr,

Thanks for reporting this important issue.
Unfortunately, I don't own such a monitor, but I think I know why it happens.
I would need your help in debugging it if that sounds reasonable to you.
I can point you to the exact location of the issue.
Please let me know what you think :)

**Comment by rodrigojmr** [comment]
Hey @idoshamun, of course! Go ahead.

**Comment by idoshamun** [comment]
I think the problem here is the positioning of the trigger for the infinite scroll.
You can check the settings here:
https://github.com/dailydotdev/daily-apps/blob/master/packages/extension/src/routes/Home.vue#L877
There are two ways to tackle it:
- Set-up a local environment to debug this issue while tweaking the `bottom` value of the anchor element
- Open Daily's New Tab and navigate to the anchor element, then try to tweak the `bottom` value.

Once you find something stable we can apply this rule to large screens using a media query.
Let me know what you think

**Comment by rodrigojmr** [comment]
I couldn't build the local environment, I had this error on `npx lerna run build`:

```
lerna ERR! yarn run build stderr:
error An unexpected error occurred: "Cannot create property 'module' on string '{\"module\":\"C:\\\\Users\\\\rodri\\\\.npm-init.js\"}'".
error Command failed with exit code 1.

lerna ERR! yarn run build exited 1 in '@daily/components'
lerna WARN complete Waiting for 1 child process to exit. CTRL-C to exit immediately.
npm ERR! code 1
npm ERR! path M:\Coding\projects\open-source\daily-apps
npm ERR! command failed
npm ERR! command C:\WINDOWS\System32\cmd.exe /d /s /c lerna run build
```

Up until then it was fine. But besides that, I tried tweaking the `bottom` value, and changing to `102%` worked. Had to scroll up and then down, but it's probably because I changed the value after the page loaded. As you can see, it loaded sometimes before changing the value, so it's very weird behavior.

[Video.](https://streamable.com/24ep5d)

**Comment by idoshamun** [comment]
I searched for this error and some say that it might have something to do with your global or local npmrc file.
https://github.com/sockethub/sockethub/pull/250
Could you please check it? Once we resolve this error it will be easier to debug.
If it won't work I'll send you a zip file with the new value so you can check it.

**Comment by rodrigojmr** [comment]
I've looked everywhere for solutions for this npmrc file issue. I've deleted all npmrc files I have, checked the npm local and global config list and they're all default. I don't know if I should report this to lerna or yarn but I can't find anyone else who's had the same issue relating to .npm-init.js on the user folder. This is my current .npm-init.js: 

```module.exports = {
    version: '0.1.0',
    name: name,
    main: 'index.js',
    scripts: {
        start: 'node index.js'
    }
}```

Unless we can solve this, I'd need the zip file.

**Comment by idoshamun** [comment]
Let's proceed with a zip file. I think it's easier, I don't want to hold you.
The zip file contains the latest version of our Chrome extension.
Unzip it and load it via chrome://extensions/ (remember to enable developer mode).
You can find the `#anchor` css in `css/home.23f199d4.css`.
Once you change the `bottom` property you can just refresh the extension page and it should reload from the filesystem.
Let me know if I missed anything.

**Comment by rodrigojmr** [comment]
Unless I'm missing something I don't think the zip file was attached 😅

**Comment by idoshamun** [comment]
Yap, sorry about that.
[extension-v2.29.0-src.zip](https://github.com/dailydotdev/daily/files/5832169/extension-v2.29.0-src.zip)

**Comment by rodrigojmr** [comment]
Thanks! So yeah, so far starting at 102vh works very consistently, and higher values,(let's say 110vh, but maybe more) , not having a noticeable difference. Regardless of the value, sometimes if scroll quite fast down, sometimes it will get stuck, but a reload and scrolling slightly slower fixes. I assume this will have something to do with the more intense loading of articles and images in a wider screen.

**Comment by idoshamun** [comment]
Thanks for checking! I'll test the 102vh setting on smaller screens to see if we can have one unified setting for all screens.
What is exactly getting stuck? Is it the whole application or just the infinite scroll?

**Comment by rodrigojmr** [comment]
The infinite scroll, yes. It doesn't load more articles and the 3 loading dots don't appear at the bottom.

**Comment by idoshamun** [comment]
Oh, that's a bummer. Might be something to do with the intersection observer.
I'm actually implementing this for the web app now. So once it will be ready there we can check out how if it works fine there.

**Comment by idoshamun** [comment]
Could you please let me know how the feed here is working for you?
https://app.daily.dev

**Comment by rodrigojmr** [comment]
I think the app is working well! In terms of scrolling, it doesn't get stuck. Sometimes it'll load just 9 articles when I switch categories (doesn't seem to do it in 1080p), but that's it.

As a sidenote, I was able to change to WSL and build from the extension's source code as per the instructions, but I don't think I can log in as maybe it requires the Daily Redirector, so I couldn't text the bookmarks view.

**Comment by idoshamun** [comment]
That sounds great that it works well! We're planning on using some of the webapp code in the extension because it's much better. So in the long term, you can expect a full solution to your issue and in the short term, I think I'll release a version with the anchor set to `102vh` or even `110vh` whatever works for you better. The 9 articles issue is because there's a bug I guess with the responsive loading, I'll look into it.

Regarding the login,  you need to have a webapp instance running. I forgot to mention it in the docs as it's a pretty new thing. We moved all the login and registration process to the webapp.

Anyway, your insights are super helpful to me :)

Let me just know what works better in terms of anchor positioning so I can proceed with a new release.

**Comment by rodrigojmr** [comment]
Hey, apologies for the delay. I can't say with certainty that notice a difference between `102vh` and `110vh`. Perhaps just go with `110vh`!

**Comment by idoshamun** [comment]
Ok, perfect. That's what I'll do! Thanks for everything. I'll let you know once it's released.

**Comment by idoshamun** [comment]
I've published a new version to the store v2.29.1. It should be available soon.
Thank you so much for helping me out on this one 🙏
